src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const ulList = [...document.querySelector('ul').children];
```

Comment:
it's a good practice to add ref/el suffix to variables with a reference to a node element
```suggestion
const ulListRef = [...document.querySelector('ul').children];
```